solutions/scripts/delayed.js

```diff
@@ -20,18 +20,19 @@ import { loadBreadcrumbs } from './breadcrumbs.js';
 sampleRUM('cwv');
 
 const LANGUAGE_COUNTRY = getLanguageCountryFromPath(window.location.pathname);
-const LAUNCH_URL = 'https://assets.adobedtm.com';
+// const LAUNCH_URL = 'https://assets.adobedtm.com'; */
```

Comment:
@cuzuco2 The code here needs to be exactly as it was before. The change of URL should be done in the placeholder file on SharePoint. Here are the docs: https://www.aem.live/developer/placeholders

We'll fix the Ghost Inspector later. It has been broken for a while and is sensitive to changes because it relies on images for comparison.